src/System.Private.Xml.Linq/src/System/Xml/Linq/XCData.cs

```diff
@@ -49,6 +52,27 @@ public override void WriteTo(XmlWriter writer)
             writer.WriteCData(text);
         }
 
+        /// <summary>
+        /// Write this <see cref="XCData"/> to the given <see cref="XmlWriter"/>.
+        /// </summary>
+        /// <param name="writer">
+        /// The <see cref="XmlWriter"/> to write this <see cref="XCData"/> to.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// The CancellationToken to use to request cancellation of this operation.
+        /// </param>
+        /// <returns>
+        /// A Task that represents the eventual completion of the operation.
+        /// </returns>
+        public override Task WriteToAsync(XmlWriter writer, CancellationToken cancellationToken)
+        {
+            if (writer == null)
+                throw new ArgumentNullException("writer");
```

Comment:
Nit: `nameof(writer)` (I think we switched over to using `nameof` throughout after these changes were already checked-in to the future branch). Applies to a few other files in this PR.